build.bat

```diff
@@ -106,8 +106,6 @@ set BJAM_OPTIONS=toolset=%BJAM_TOOLSET%^
 set BJAM_OPTIONS_X64=%BJAM_OPTIONS%^
  address-model=64^
  --stagedir=stage_x64^
- define=BOOST_USE_WINAPI_VERSION=0x0501^
- cxxflags="/Zc:threadSafeInit- "
 

```

Comment:
Well, Windows Server 2003 x64 (including Windows XP x64) should not be ignored. `BOOST_USE_WINAPI_VERSION` should therefore be `0x0502` here.